src/atoms/Flex/__tests__/Flex.native.test.js

```diff
@@ -1,15 +1,140 @@
 import React from 'react';
 import { render } from '@testing-library/react-native';
-import { View } from 'react-native';
+
+import View from '../../View';
 import Flex from '../index';
+import Space from '../../Space';
+import Size from '../../Size';
+
+const defaultFlexStyles = { flexDirection: 'column' };
+const defaultSpaceStyles = {};
+const defaultSizeStyles = {};
 
 describe('Renders <Flex /> correctly', () => {
-  it('snapshot testing', () => {
-    const { container } = render(
+  it('renders child of <Flex />(no styles) correctly', () => {
```

Comment:
You don't have to use <Flex /> everytime as `<Flex />` as already in `describe("<Flex />")`